lib/errors.js

```diff
@@ -28,6 +28,9 @@ module.exports = function(Sequelize) {
 		},
 		'>=4.0.0': function() {
 			return require('./errorsEs6')(Sequelize);
+		},
```

Comment:
semver-select will select >=4.0.0 in case of sequelize5 because it selects first choise that matches according to documentation